content/docs/explanation/netzgrafik-editor/_index.en.md

```diff
@@ -7,75 +7,35 @@ description: "Open-source software developed by SBB CFF FFS and its integration
 
 Netzgrafik-Editor (NGE) is an open-source software that enables the creation, modification, and analysis of regular-interval timetable, at a macroscopic level of detail, developed by [Swiss Federal Railways (SBB CFF FFS)](https://www.sbb.ch/). See [front-end](https://github.com/SchweizerischeBundesbahnen/netzgrafik-editor-frontend) and [back-end](https://github.com/SchweizerischeBundesbahnen/netzgrafik-editor-backend) repositories.
 
-OSRD (microscopic level of detail, trains scheduled once, based on a defined infrastructure, depicts a timetable) and NGE (macroscopic level of detail, regular-interval based train runs, no infrastructure, depicts a transportation plan) are semantically different, but close enough to make it work together.
+OSRD (microscopic level of detail, based on a in-quality defined infrastructure, depicting a timetable composed of unique train schedules) and NGE (macroscopic level of detail, based on no explicit infrastructure, depicting a transportation plan composed on regular-interval based train runs) are semantically different, but close enough to make it work together.
```

Comment:
"in-quality defined infrastructure" sounds a bit weird. Maybe "well-defined infrastructure"? Or "high-quality infrastructure"?

"based on no" sounds weird as well. "not based on any"?

"composed on": the preposition is wrong here. "made up of" would sound more natural.

Maybe instead of the super long parentheses before the main point, a different sentence structure could be used? e.g. "OSRD and NGE data models are semantically different: the former uses a […], while the latter uses a […].".